cmd/soroban-rpc/internal/methods/get_transaction_test.go

```diff
@@ -154,6 +154,7 @@ func TestGetTransaction(t *testing.T) {
 		ResultMetaXdr:         expectedTxMeta,
 		Ledger:                101,
 		LedgerCloseTime:       2625,
+		DiagnosticEventsXDR:   []string{},
```

Comment:
can you add a test case here where we exercise the case in which events are included in the response?